sdk/communication/AzureCommunication/Source/Identifiers.swift

```diff
@@ -116,13 +116,20 @@ import Foundation
      - Parameter id: Full id of the Microsoft Teams user.
      - Parameter cloudEnvironment: The cloud that the Microsoft Team user belongs to. A null value translates to the Public cloud.
      */
-    public init(userId: String, isAnonymous: Bool = false, identifier: String? = nil, cloudEnvironment: CommunicationCloudEnvironment = CommunicationCloudEnvironment.Public) {
+    public init(userId: String, isAnonymous: Bool = false, identifier: String? = nil, cloudEnvironment: CommunicationCloudEnvironment = .Public) {
         self.identifier = identifier
         self.userId = userId
         self.isAnonymous = isAnonymous
         self.cloudEnviroment = cloudEnvironment
     }
 
+    public init(userId: String, isAnonymous: Bool) {
```

Comment:
@raosanat I added a new init to fill the roll of default parameters to make it a little easier in Obj-C